sui/src/config/mod.rs

```diff
@@ -69,7 +69,7 @@ pub struct AuthorityInfo {
     pub base_port: u16,
 }
 
-#[derive(Serialize, Debug)]
+#[derive(Serialize, Debug, Clone)]
```

Comment:
It seems this is no more the authority private info, since it has no private key? Traditionally we have tried to keep stuff with private keys non-Clone, but this is not an issue any more?

Reply:
Agreed. Will rename

Reply:
we should further clean "private" info out such as the `db_path`